File: C_Programming/Competitive/Factor.c


void DisplayFactor(int ino)
{
    int i=0;
    if (ino<=0)
    {
        ino= -ino;
    }
    

    for(i=1;i<ino;i++)
    {
        if ((ino%i==0) && (i%2 ==0 ))
        {
             printf("%d\n",i);
            
        }
        
       
    }

}

int main()
{
    int iValue=0;
    
    printf("Enter the no:");
    scanf("%d",&iValue);

    DisplayFactor(iValue);


    return 0;

}

File: Java_Programming/Competitive/Factor.java

import java.util.Scanner;


class Factor
{
    public static void main ( String A[])
    {
        Scanner sobj= new Scanner(System.in);

        int iValue=0;

        System.out.println("Enter the Value: ");
        iValue = sobj.nextInt();
        
        Logic lobj = new Logic();

        lobj.DisplayFactors(iValue);


      

       

    }
    

}

class Logic
{
    void DisplayFactors(int iNo)
    {
        int iCount=0;

        System.out.println("Factors of "+iNo);
        for(iCount = 1; iCount <= iNo; iCount++)
        {
            if(iNo%iCount==0)
            {
                System.out.println(iCount);
                
            }
           
           
        }
        
        
    }

}

